src/utils/objectToUrlParams.ts

```diff
@@ -4,7 +4,7 @@ interface UrlDataProps {
 
 export default function objectToUrlParams(data: UrlDataProps) {
   const params = Object.entries(data)
-    .filter(([, value]) => value !== undefined && value !== null)
+    .filter(([, value]) => !!value)
```

Comment:
Should not need it...

```suggestion
    .filter(([, value]) => value)
```